CHANGELOG.md

```diff
@@ -14,6 +14,9 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 - Added support for Estonian 🇪🇪 It currently includes the gold-standard Estonian Valence
   sentiment classification dataset. The split is given by 1,024 / 256 / 2,048 samples
   for train / val / test, respectively. This was contributed by @slowwavesleep ✨
+- Added support for the gold-standard Estonian Winogrande dataset. The split is given by
```

Comment:
Can you merge this entry with the previous one, to have a single one with the Estonian addition?  [Here](https://github.com/EuroEval/EuroEval/releases/tag/v15.12.0) is an example from when Portuguese was added.